src/t2iapi/alert/service.proto

```diff
@@ -76,4 +76,15 @@ service AlertService {
     Initiate alarm off for the AlertCondition with the given handle.
      */
     rpc InitiateAlarmOff (BasicHandleRequest) returns (BasicResponse);
+
+    /*
+    For the AlertSignal with the provided handle do the following:
+     - ensure that prior to the following settings the pm:AlertSignalState/@Presence is set to "On"
+     - set the pm:AlertSignalState/@ActivationState to "On" and at the same time
+     - change the associated pm:AlertConditionState/@Presence to "false"
+    Ensure that the @ActivationState remains set to "On" for as long as the the associated AlertCondition @Presence
+    changes to "false".
```

Comment:
To me it is hard to differentiate between this and the next without any context. Can the `@ActivationState` already be `On` or does it have to change to on. Can the `pm:AlertConditionState/@Presence` already be false or does it have to change to false from a different value? I think set and change is supposed to imply a difference, but this should really be written explicity. Maybe we need to express this more explicitly, a suggestion we could workshop would be

```suggestion
    For the AlertSignal with the provided handle ensure that the following sequence appears
    - State 1
       - pm:AlertSignalState/@Presence must be "On"
       - pm:AlertSignalState/@ActivationState must be "On"
       - the associated pm:AlertConditionState/@Presence must be "true"
    - State 2
       - pm:AlertSignalState/@ActivationState must be "On"
       - the associated pm:AlertConditionState/@Presence must be "false"
```